tools/preview/CHANGELOG.md

```diff
@@ -4,12 +4,140 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 <a name="3.0.22"></a>
-## 3.0.22
+## 3.0.22 [Upgrade to v7]
```

Comment:
This is great though we don't typically update our changeling manually. It's built by our tooling from the commit message(s).